extensions/shared/blocks/helpers/helpers.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
Let's put this right into the `extensions/shared` directory: `extensions/shared/get-category-with-fallbacks.js` (or add something block-specific in the name if you like).

Reply:
Sounds good :+1:

Reply:
Actually, why not adding this same file, but to `extensions/shared/blocks`?

Reply:
I don't know why the `blocks` subdirectory was added or whether it serves much organizational purpose. There seems to be block-specific functionality in `extensions/shared`.

If you have reason to believe it's justified and preferable to put it under `blocks`, I won't object.

Reply:
As in `extensions/shared/blocks/get-category-with-fallbacks.js`.

Reply:
You're right, the `extensions` folder already seem to be mainly about the blocks. I'll just add there as you suggested. Thanks!

Reply:
> I don't know why the blocks subdirectory was added or whether it serves much organizational purpose.

That directory is used for modifications of single, non-Jetpack, blocks today, such as the Cover block on WordPress.com.

Since your changes will impact multiple blocks, I would recommend having your changes in `extensions/shared`.